packages/core/src/components/discord-mention/DiscordMention.ts

```diff
@@ -85,10 +88,32 @@ export class DiscordMention extends LitElement implements LightTheme {
 
 	/**
 	 * The type of mention this should be. This will prepend the proper prefix character.
-	 * Valid values: `user`, `channel`, `role`, `voice`, `locked`, `thread`, `forum`, and `slash`.
+	 * Valid values:
+	 * - `'user'`
+	 * - `'channel'`
+	 * - `'role'`
+	 * - `'voice'`
+	 * - `'locked'`
+	 * - `'thread'`
+	 * - `'forum'`
+	 * - `'slash'`
+	 * - `'server-guide'`
+	 * - `'channels-and-roles'`
+	 * - `'customize-community'`
```

Comment:
it would make more sense to go with the official mentions imo
<id:guide>
<id:customize>
<id:browse>

Reply:
I don't like those. I rather went for how they show up in the client, not what they are known as internally. That's also why it's called `channels-and-roles` and not `browse-channels` because technically both are possible when you type `#!` in chat but they go to the same channel, but a different tab.

Reply:
Ah fair

Reply:
I realise I dropped half a sentence on the `channel-and-roles` reasoning so to further clarify, the reason I chose it over `browse-channels` is because it shows as `Channels & Roles` in the client at the top of the channel list and only from there is `Browse Channels` a possible tab.

Reply:
No, it depends on the configuration. If self roles aren't srt up, it only shows browse channels.

Browse channels on the other hand is always active even if self roles are missing.

It's correct tho that it moves in the channel & roles tab if both is enabled.